### ISSUE TITLE
Is the SA-CLIP model used?

### ISSUE THREAD

**Issue by Mangata14** [jiangyitong/AutoDIR#17]
Hello,

It seems that your code does not use the SA-CLIP model that you proposed. I did not see the calculation of the cosine similarity between image embeddings and text embeddings, but instead, the CLIP model is used directly. This appears to be inconsistent with what is stated in your paper. Am I misunderstanding something, or is there another reason for this?

I look forward to your reply, and wish you all the best in your work.

**Comment by jiangyitong** [comment]
Hi! Thanks for asking, SA-CLIP and CLIP have the same inference code, and the training loss is not included in this repo now. I will release the training code for SA-CLIP soon

**Comment by Mangata14** [comment]
> Hi! Thanks for asking, SA-CLIP and CLIP have the same inference code, and the training loss is not included in this repo now. I will release the training code for SA-CLIP soon

OK，thanks for your fantastic work again.
